sig-network/annual-report-2021.md

```diff
@@ -0,0 +1,63 @@
+    - Meetings generally start with 15 minutes of group issue or PR triage, followed by 45 minutes of topics from the agenda, which are crowd-sourced throughout the week leading up.
+    - Recordings are automatically uploaded to the YouTube channel and are up to date.
+    - Notes are stored in the minutes document, which is also where topics can be added to the agenda in advance.
+
+- How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?
+  - Updates from subprojects are generally the responsibilitiy of subproject members themselves (i.e., push not pull).
+  - Updates are usually on the mailing list, occasionally subprojects will also use meeting time to discuss in real-time.
+- Same question as above but for working groups.
+  - Same answer as above, but for working groups.
+- When was your last monthly community-wide update? (provide link to deck and/or recording)
+  - Last community meeting was long enough ago that it is no longer in the minutes history (so, clearly we're due).
+
+# Membership
+
+- Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+  - Yes
+- How do you measure membership? By mailing list members, OWNERs, or something else?
+  - We don't currently have an explicit way of measuring membership.
+- How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?
+  - We don't currently have an explicit way of measuring reviewer / approver bandwidth. Everyone feels overburdened, we're not doing anything about it.
+- Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?
+  - The easiest way for new contribs to get involved is to attend a meeting and volunteer to help triage an issue during our triage session.
+- What programs do you participate in for new contributors?
+  - No programs specifically targeting new contribs at the moment.
+- Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?
+  - Yes - we have long-term as well as one-time contributors from a variety of companies. 
+
+# Current initiatives and project health
+
+> Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members. 
+
+- What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?
+- Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?
+  - EndpointSlice: [GA](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/0752-endpointslices)
```

Comment:
gateway reached v1alpha1

cc" @robscott

Reply:
Good call, TopologyAwareHints also reached alpha

Reply:
If this is covering 2020, we should also probably mention that Ingress reached GA.